### DISCUSSION TITLE
Update함수가 Start함수보다 먼저 호출됩니다.

### DISCUSSION THREAD

**Discussion by Cha-SeongHun** [naverz/zepeto-studio-kor#657] [Issue Report (Legacy)]
패키지 버전은 1.2.0입니다.
재현방법은 저도 모르겠습니다 일부 스크립트에서만 발생하는군요

코드는 아래와 같습니다.
```ts
 private flag: bool = true;
 
 private Start() {
    console.log(`[Start] ${this.name}`);
  }

  private Update() {
    if (this.flag) {
      this.flag = false;
      console.log(`[Update] ${this.name}`);
    }
  }
```

이를 실행하면 아래와 같이 로그가 발생합니다.

![image](https://user-images.githubusercontent.com/100827088/165060814-c79ddd0a-d64e-4a21-9c80-55572164cfb1.png)

영향이 큰 버그라 우선순위 높게 확인해주시면 감사하겠습니다!

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요, 내부에서 확인 되어 다음 업데이트시 반영될 예정입니다.
불편드려 죄송합니다.
감사합니다.
